R/readers.R

```diff
@@ -309,4 +309,29 @@ read_s3 <- function(file){
   data <- readr::read_log(file = file, col_types = types, col_names = names)
   data$request_time <- readr::parse_datetime(data$request_time, format = "%d/%b/%Y:%H:%M:%S %z")
   return(data)
-}
\ No newline at end of file
+}
+
+#'@title Read Microsoft IIS Logs
+#'@description \code{read_iis} provides a reader for Microsoft IIS access logs.
+#'
+#'@param file the full path to the IIS log file you want to read.
+#'
+#'@examples
+#'# Using the inbuilt testing dataset
+#'iis_data <- read_iis(system.file(file.path("extdata", "iis.log"), package = "webreadr"))
+#'
+#'@export
+
+read_iis <- function(file){
+  #read in the text file to search for server restarts
+  raw_file <- readLines(file)
+  mask <- !substr(raw_file, 1, 1)=="#"  
+  
+  names <- c("date", "time", "s-ip", "cs-method", "cs-uri-stem", "cs-uri-query", "s-port", 
```

Comment:
I'd suggest underscores instead of dashes so it works nicely with column access.